src/azure-cli/azure/cli/command_modules/hdinsight/_params.py

```diff
@@ -69,6 +69,8 @@ def load_arguments(self, _):
         c.argument('esp', arg_group='Cluster', action='store_true',
                    help='Specify to create cluster with Enterprise Security Package. If omitted, '
                         'creating cluster with Enterprise Security Package will not not allowed.')
+        c.argument('minimal_tls_version', arg_type=get_enum_type(['1.0', '1.1', '1.2']),
```

Comment:
Will your server have a default value for this parameter? or do you want to define a default value here?

Reply:
> Will your server have a default value for this parameter? or do you want to define a default value here?

Thanks for your comment.
Yes, our server have a default value for this parameter, and customer doesn't have to specify this parameter I mean it can be null. I don't want to define a default value here.